### ISSUE TITLE
Control-click on Mac OS X result in all subsequent clicks reported as right-click

### ISSUE THREAD

**Issue by mcslee** [processing/processing#5765]
## Description
Performing a ctrl-click on Mac OS X results in the click button being reported as RIGHT, which is correct. However, it sets a flag which is never unset, such that *all* subsequent clicks in the operation of the program are reported as right-clicks, for the rest of the program's operation - whether or not the control button is down.

## Expected Behavior
Only clicks performed while the control modifier key is actively down should be reported as a RIGHT click.

## Current Behavior
Every click after the first control-click is reported as button = RIGHT.

## Steps to Reproduce
```
void setup() {
  registerMethod("mouseEvent", this);
}

void draw() {
}

void mouseEvent(MouseEvent mouseEvent) {
  if (mouseEvent.getAction() == MouseEvent.PRESS) {
    println(mouseEvent.getButton());
  } 
}
```

Run the above and observe that after a control-click, on Mac OS, all future events are reported as RIGHT button.

## Your Environment
MacOS Mojave 10.14.2
Processing 3.5.1

## Possible Causes / Solutions
The issue is caused by this line:
https://github.com/processing/processing/commit/2d2b4779013045512cb469e615b0dc376672292a#r32029420

**Comment by benfry** [comment]
Ouch; thanks for finding that.

**Comment by mcslee** [comment]
Sure thing - appreciate the quick merge!

**Comment by benfry** [comment]
Fixed with #5766 for Processing 3.5.2.
